client/web/src/components/series/label/SeriesLabel.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useDispatch } from 'react-redux'
+import {
+  Chip,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  Box,
+} from '@material-ui/core'
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+
+import { actionTypes, SeriesService } from '../../../features/series'
+import { ILabel } from '../../../models/ILabel'
+import { ISeries } from '../../../models/series'
+import { SeriesLabelFrom } from './SeriesLabelFrom'
+
+type SeriesLabelProps = { series: ISeries }
+
+export const SeriesLabel: React.FC<SeriesLabelProps> = (props: SeriesLabelProps) => {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+  let labels = props.series.labels;
+
+  // For form
+  const [open, setOpen] = React.useState(false);
+  const handleClickOpen = (localSeries: ISeries | null) => {
+    setOpen(true);
+    dispatch({ type: actionTypes.SET_CURRENT_SERIES, series: localSeries })
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleDelete = (localSeries: ISeries | null, labelClicked: ILabel) => {
+    if(localSeries == null)
+    {
+      throw Error("No series selected")
+    }
+    if(localSeries.labels == null)
+    {
+      localSeries.labels = new Array<ILabel>();
+    }
+    localSeries.labels.forEach((label: ILabel) => {
```

Comment:
Why do you do a forEach here?

You just need next line to remove label

```typescript
localSeries.label = localSeries.labels.filter((label: ILabel) => label.id !== labelClicked.id)
```

Reply:
Done